### DISCUSSION TITLE
modulo 8 kata 8

### DISCUSSION THREAD

**Discussion by LuisAntonioLinaresCarbajal** [LaunchX-InnovaccionVirtual/CursoIntroPython#211] [Q&A]
hola, ¿Alguien ya pudo realizar la kata 8 ejercicio 2 ? llevo todo el dia peleándome con ese ejercicio ¿como sumo los valores de una lista 
?

**Comment by OscarPedroza** [comment]
Hola, para sumar los valores de la lista 'moons' se debe iterar la lista en un ciclo For. y los diferentes cantidades de las lunas se las vas sumando y asignando a una variable que guarde el total de lunas.

Sería algo como: `total_moons += moon`  donde moon es la variable que recorre la lista

**Reply by doguedogue** [reply]
Se podría con:
- LISTA[0] + LISTA[1] +...
- Ciclo for
- Ciclo while 
- un reduce

**Reply by Oscar19260** [reply]
Correcto haces el ciclo for y vas realizando la suma conforme va iterando, algo como:
```
for moon in range of moons size
    total moons +=  moon
```

**Comment by Dfredude** [comment]
Ahi te va un hack:
```python
cantidad_de_lunas = sum(planet_moons.values())
```
- `planet_moons.values()` te regresa la lista de numeros de todas las lunas que hay en cada planeta, justo como quieres.
- `sum(sequence)` es una funcion que suma los valores de cualquier sequencia de python. En este caso la usamos con una lista: `sum(planet_moon.values())` y nos devolvera la suma de todos los valores la cual es **_214_**.
### Espero haberte ayudado!
